tools/c7n_gcp/c7n_gcp/query.py

```diff
@@ -412,7 +412,12 @@ class TypeInfo(metaclass=TypeMeta):
     refresh = None
 
     @classmethod
-    def get_metric_resource_name(cls, resource):
+    def get_metric_resource_name(cls, resource, metric_key=None):
```

Comment:
thanks, but the override for this should be on compute resources  resource type (reosurces/compute.py:Instance.resource_type) rather than the base type class, the signature in the base class should change, but otherwise unaffected.